scripts/import_organisation.rb

```diff
@@ -0,0 +1,21 @@
+# bundle exec rails runner scripts/import_organisation.rb eer53d
```

Comment:
I would add `bundle exec rake db:create db:schema:load db:migrate` or a more appropriate call to highlight the prerequisite.

Reply:
Actually why ddi you crated this routine? I would think that `pg_restore` would be enough to get a working instance of RDV-Solidarités with the data subset.

Reply:
we could try to use `rake db:create` instead of `createdb`, yes. however `db:migrate` is not useful because `db:schema:load` already loads the latest schema.

this import routine is useful to import the `csv` that were output before. I don't think that `pg_restore` is able to import `csv` files that were output by `COPY`. I think you need to use `COPY` statements to import them as well.  I used `COPY` instead of `pg_dump` because `pg_dump` doesn't allow for filter queries. `COPY` has the additional advantage of creating CSVs which we could also give as part of the export for agents who don't know how to import a sql dump.

we could stop needing ruby routines and relying on the gem by simply using COPY commands, I think it'd work rather easily.